.github/workflows/build.yml

```diff
@@ -117,6 +125,14 @@ jobs:
         cp am/attract* "${GITHUB_WORKSPACE}"/artifacts/
         7z a "attract-${{matrix.platform.name}}-${{matrix.config.name}}-x86_64.7z" artifacts
 
+    - name: Prepare macOS artifacts
+      if: runner.os == 'macOS'
+      run: |
+        mkdir -p "${GITHUB_WORKSPACE}"/artifacts
+        chmod +x am/util/osx/appbuilder.sh
+        bash am/util/osx/appbuilder.sh
```

Comment:
The best would be to push the file already chmodded. If you can't, no need of the chmod, just keep the 2nd line

Reply:
Removed the chmod line, kept the bash command.